packages/components/src/ListView/Items/Items.component.js

```diff
@@ -55,6 +84,20 @@ export class ItemsComponent extends React.PureComponent {
 		return this.props.items.length;
 	}
 
+	oldGetRowHeight({ index }) {
+		if (this.hasToggleAll && index === 0) {
+			return 40;
```

Comment:
`TOGGLE_ALL_ROW_HEIGHT `

Reply:
Left old code in place, let me change that